libpod/container_internal_linux.go

```diff
@@ -1505,6 +1506,17 @@ func (c *Container) getOCICgroupPath() (string, error) {
 	case (rootless.IsRootless() && !unified) || c.config.NoCgroups:
 		return "", nil
 	case c.runtime.config.Engine.CgroupManager == config.SystemdCgroupsManager:
+		if c.config.CgroupsMode == "no-conmon-delegated" {
```

Comment:
Can we make "no-conmon-delegated" a constent somewhere?

Reply:
> The commit messages states `conmon-delegated` but the code adds a `no-conmon-delegated`. I'd guess we should update the docs as well.

Thanks, I've renamed it to `conmon-delegated`

> Do we need to update the systemd unit files in `pkg/system/generate`?

I don't think we can do that yet.  crun needs https://github.com/containers/crun/pull/409